test/benchmarks/material/button/button.perf-spec.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {$, browser} from 'protractor';
+import {runBenchmark} from '@angular/dev-infra-private/benchmark/driver-utilities';
+
+describe('button performance benchmarks', () => {
+  beforeAll(() => {
+    browser.rootEl = '#root';
+  });
+
+  it('renders a basic raised button', async() => {
+    await runBenchmark({
+      id: 'button-render',
+      url: '',
+      ignoreBrowserSynchronization: true,
+      params: [],
+      setup: async () => await $('#show').click(),
```

Comment:
Could we remove `setup` here, or do I miss something? Looks like we first click `show` and then for subsequent sampling iterations we call `#hide` before and then measure `#show`.